vyper/functions/functions.py

```diff
@@ -54,6 +55,11 @@
 )
 
 
+SHA256_ADDRESS = 2
+SHA256_BASE_GAS = 60
+SHA256_PER_WORD_GAS = 6
```

Comment:
Not sure about this? My reading of the yellow paper says sha256 per word is 12. Seems to be confirmed by https://github.com/ethereum/py-evm/blob/d6adf9039647b6fc39b9d79725e9331008a5c438/eth/constants.py#L85

Reply:
Ah right, I must've used the wrong source/reference :+1: